modules/reference/pages/feature/mpJwt/examples.adoc

```diff
@@ -42,7 +42,7 @@ You can configure a truststore to verify JSON Web Tokens. The default keystore f
 [source,xml]
 ----
 <ssl id="defaultSSLConfig" keyStoreRef="defaultKeyStore" trustStoreRef="defaultTrustStore" />
-<keyStore id="defaultTrustStore" location="${exampleconfigvariable}" type="JKS" password="changeit" />
+<keyStore id="defaultTrustStore" location="MyKeyStoreFile.jks" type="JKS" password="changeit" />
```

Comment:
since .jks is an older file format, we should use .p12, which is liberty default. So `location="MyKeyStoreFile.p12"`